### ISSUE TITLE
/admin/donations should have "send reciept" button

### ISSUE THREAD

**Issue by echarlie** [wuvt/donormotor#14]
the donations page should allow one to automatically send a receipt, if the donation was "ordered" at least 30 days ago, paid, premiums (if available) were shipped, and a valid email address is available.

else, it should allow printing of a receipt.

**Comment by mutantmonkey** [comment]
Yes, the Stripe receipts are different and don't include tax information.
